### PR TITLE
Night vision goggles no longer cancel out thermal eyes

### LINKED ISSUE #35921: NVGs overwrite Thermal vision implants
Night vision goggles overwrite thermal vision implants.

[Reproduction]: get a thermal implant in nuke op uplink, use, put on NVG goggles, thermal vision disapears.

### PR DESCRIPTION
**Pull request by ghost**
:cl:
fix: night vision goggles no longer overwrite thermal eyes. nvg eyes no longer overwrite thermal goggles.
/:cl:

fixes #35921

### CODE REVIEW

**Comment by optimumtact** [comment]
Welcome to the repo

**Comment by iskyp** [comment]
ty

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -109,7 +109,6 @@
 	icon_state = "night"
 	item_state = "glasses"
 	darkness_view = 8
-	vision_flags = SEE_BLACKNESS
```
Comment (ChangelingRain) [inline code/modules/clothing/glasses/_glasses.dm]: #31114

Comment (Jalleo) [reply code/modules/clothing/glasses/_glasses.dm]: WELP

Comment (iskyp) [reply code/modules/clothing/glasses/_glasses.dm]: I've tested this on both 511 and 512 without any issues, although I'm looking into it at the moment

Comment (iskyp) [reply code/modules/clothing/glasses/_glasses.dm]: so after going through many older versions of the code, I've figured out it was probably something to do with parallax.
I haven't exactly found what commit started space-vision, nor what commit fixed it, but it's definitely not a problem anymore.
going to push another commit for nvg eyes and then go to bed will post pics of proof this works fine tmrw if requested

Comment (vuonojenmustaturska) [reply code/modules/clothing/glasses/_glasses.dm]: please post pictures, i'm a little hesitant to essentially revert antur's fixes

Comment (AnturK) [reply code/modules/clothing/glasses/_glasses.dm]: This is *probably* fine the real fix was moving the game plane from 0.

**Pull request merged**